futaba/cogs/reloader/core.py

```diff
@@ -230,12 +232,27 @@ def unload_cog(self, cogname, check_missing=True):
     async def listcogs(self, ctx):
         """ Lists all currently loaded cogs. """
 
-        content = StringBuilder("```yaml\nCogs loaded:\n")
-        if self.bot.cogs:
-            for cog in sorted(self.bot.cogs):
-                content.writeln(f" - {cog}")
-        else:
-            content.writeln(" - (none)")
+        content = StringBuilder("```\n")
+
+        extensions = {}
+        for cog_name, cog in self.bot.cogs.items():
+            ext_name = cog.__module__.rsplit('.', 1)[0]
+            ext_name = ext_name.rsplit('.', 1)[1]
```

Comment:
Why is this split twice? Is there a possibility of using `os.path.splitext` or something?

Reply:
I'm splitting it twice as we need the last part of the path and not the file name e.g futaba.cogs.**info** and not futaba.cogs.info.**core**

I could use `os.path.splitext` however we would still need a split
`ext_name, file_name = splitext(cog.__module__)`
`ext_name = ext_name.rsplit('.', 1)[1]`